common/controllers/ResourceController.ts

```diff
@@ -3,13 +3,17 @@ import { Resource } from '../../mobile/src/constants/Resource';
 
 export default class ResourceControllerBase {
 
-  public async getResources(strategySlug: string): Promise<Resource[]> {
+  public async getResourcesByStrategy(strategySlug: string): Promise<Resource[]> {
     return await RepoFactory.Instance.resources.getByStrategySlug(strategySlug);
   }
 
-  public async getMultipleResources(strategySlugs: string[]): Promise<Resource[]> {
+  public async getMultipleResourcesByStrategies(strategySlugs: string[]): Promise<Resource[]> {
```

Comment:
This is better named `getResourcesByStrategies` since the multiple is also applying to the other function. Same as the `getMultipleResourcesBySlugs`